llvm/include/llvm/Support/RevngOptions.h

```diff
@@ -0,0 +1,13 @@
+#ifndef LLVM_SUPPORT_REVNGOPTIONS_H
+#define LLVM_SUPPORT_REVNGOPTIONS_H
+
+#include "llvm/Support/CommandLine.h"
+
+namespace llvm::revng {
+
+extern bool OverrideRevngDebugInformationPreservationStyle;
```

Comment:
AFAIU you put this to enable `revng` to force this to `true`. Correct?

I'd avoid this and just use `TemporaryLLVMOption.h` or directly go with `getOption<bool>(Options, "option-name")->setInitialValue(...)`.

It's not the nicest but I prefer to do things in a single way.

Reply:
Done.
(pasting from the revng-side sister PR)
```cpp
    // Force-enable `--enable-strict-debug-information-preservation-style` for
    // revng binaries even if it wasn't specified.
    EnableStrictDebugInformationPreservationStyle->setInitialValue(true);
```